eva-remapping-get-source/src/test/resources/application.properties

```diff
@@ -21,3 +22,7 @@ mongodb.read-preference=primary
 
 # See https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding
 spring.main.allow-bean-definition-overriding=true
+
+# to fix exception javax.management.InstanceAlreadyExistsException: com.zaxxer.hikari:name=dataSource,type=HikariDataSource
+# see https://stackoverflow.com/a/51798043/2375586
+spring.jmx.enabled=false
```

Comment:
You could also try using [this additional property](https://github.com/sundarvenkata-EBI/eva-accession/blob/7a3fdd6b004b2dfecc6177b9c1abb3c3b5190eca/eva-accession-ws/src/main/resources/application.properties#L26-L26) so that re-deployment does not cause any headaches.